### PR TITLE
SA Issue Fixes

### PR DESCRIPTION
**Pull request by mdwhitten**
Implements changes to fix a few open issues.

### CODE REVIEW

**Comment by mdwhitten** [comment]
@smooresni This should implement all open fixes to SA modules that you have submitted. Please review when you can.

File: Source/SA/RFmx LTE/RFmxLTE.cs [hunk 1]
```diff
@@ -201,6 +199,22 @@ public static void ConfigureModAcc(RFmxLteMX lte, ModAccConfiguration modAccConf
             lte.ModAcc.Configuration.ConfigureSynchronizationModeAndInterval(selectorString, modAccConfig.SynchronizationMode, modAccConfig.MeasurementOffset, modAccConfig.MeasurementLength);
             lte.ModAcc.Configuration.ConfigureEvmUnit(selectorString, modAccConfig.EvmUnit);
         }
+        public static void SelectAndInitiateMeasurements(RFmxLteMX lte, RFmxLteMXMeasurementTypes[] measurements, AutoLevelConfiguration autoLevelConfig,
+            bool enableTraces = false, string selectorString = "", string resultName = "")
+        {
+            // Aggregate the selected measurements into a single value
+            // OR of 0 and x equals x
+            RFmxLteMXMeasurementTypes selectedMeasurement = 0;
```
Comment (smooresni) [inline Source/SA/RFmx LTE/RFmxLTE.cs]:
```suggestion
            RFmxLteMXMeasurementTypes selectedMeasurements = 0;
```
suggestion for clarity, but won't request it to be changed

File: Source/SA/RFmx LTE/RFmxLTE.cs [hunk 2]
```diff
@@ -201,6 +199,22 @@ public static void ConfigureModAcc(RFmxLteMX lte, ModAccConfiguration modAccConf
...
         }
+        public static void SelectAndInitiateMeasurements(RFmxLteMX lte, RFmxLteMXMeasurementTypes[] measurements, AutoLevelConfiguration autoLevelConfig,
+            bool enableTraces = false, string selectorString = "", string resultName = "")
+        {
+            // Aggregate the selected measurements into a single value
+            // OR of 0 and x equals x
+            RFmxLteMXMeasurementTypes selectedMeasurement = 0;
+            foreach (RFmxLteMXMeasurementTypes measurement in measurements)
+                selectedMeasurement |= measurement;
+            lte.SelectMeasurements(selectorString, selectedMeasurement, enableTraces);
```
Comment (smooresni) [inline Source/SA/RFmx LTE/RFmxLTE.cs]: I'm wondering, what if the array is empty? I really don't see people using this function to disable measurements. 
Could we consider not touching SelectMeasurements if the array is empty? That might muddle understanding a little though. I'm just considering ways to still use this function with the other measure functions.

File: Source/Common/Common.cs [hunk 3]
```diff
@@ -50,5 +21,47 @@ public enum LocalOscillatorSharingMode
...
+            public double MeasurementInterval_s;
+            public double Bandwidth_Hz;
 
+            public static AutoLevelConfiguration GetDefault()
+            {
+                return new AutoLevelConfiguration
+                {
+                    Enabled = false,
+                    MeasurementInterval_s = 10e-3,
+                    Bandwidth_Hz = 20e6
```
Comment (smooresni) [inline Source/Common/Common.cs]: This only gets used by SpecAn. I'd like to propose we remove it and add it as a required parameter to SpecAn's Initiate.

File: Source/SA/RFmx LTE/RFmxLTE.cs (hunk 2 above)
Comment (mdwhitten) [reply Source/SA/RFmx LTE/RFmxLTE.cs]: My desire would be that the user would receive an error if they have not selected any measurements. Immediately after Initiate will be called, which will fail with nothing selected. In other words, no selection should be considered a user error. 

I'd be happy leaving whatever default error will be thrown by RFmx.

**Pull request merged**